src/7-to-8/major-changes/platforms.rst

```diff
@@ -210,13 +210,20 @@ And the platform settings for these examples might be:
            # A computer with PBS, that takes local job submissions
            job runner = pbs
            hosts = localhost
+           install target = localhost
 
        [[slurm_supercomputer]]
```

Comment:
The example above uses a platform named "just_run_it" - it should be "slurm_supercomputer".